package.json

```diff
@@ -26,8 +26,16 @@
     "typings": "dist/index.d.ts",
     "main": "dist/index.module.js",
     "devDependencies": {
-        "@babylonjs/core": "~5.0.0-rc.0",
-        "@babylonjs/inspector": "~5.0.0-rc.0",
+        "@babylonjs/core": "~5.16.0",

```

Comment:
would you use `^5.16.0` instead of `~5.16.0` because Babylon.js 5 follows SemVer.